config.default.json

```diff
@@ -44,6 +44,16 @@
     "google-artifact-cr",
     "workload"
   ],
+  "CRA_COMPATIBLE_TYPES": [
+    "artifactory-cr",
```

Comment:
You're missing some types. Docker hub, ever acr gcr

Reply:
nice catch,
fixed with d056b587d1114e489422ba18baf821d1dfb28ef5